<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ETFImageSearch</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="indexButton">
         <property name="text">
          <string>Create index</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="searchButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Search image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="prtestButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PR test</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget1" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeView" name="treeView"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>659</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAlgorithm">
    <property name="title">
     <string>Indexes</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionCreate_index"/>
    <addaction name="actionSearch_image"/>
    <addaction name="actionPR_test"/>
    <addaction name="actionOptimize"/>
   </widget>
   <addaction name="menuAlgorithm"/>
   <addaction name="menuActions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRGB_Histogram">
   <property name="text">
    <string>RGB Histogram</string>
   </property>
  </action>
  <action name="actionCreate_index">
   <property name="text">
    <string>Create index</string>
   </property>
  </action>
  <action name="actionSearch_image">
   <property name="text">
    <string>Search image</string>
   </property>
  </action>
  <action name="actionPR_test">
   <property name="text">
    <string>PR test</string>
   </property>
  </action>
  <action name="actionOptimize">
   <property name="text">
    <string>Optimize</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
